R/sidebar.R

```diff
@@ -221,7 +221,8 @@ layout_sidebar <- function(
       "--bslib-sidebar-max-height-mobile" = max_height_mobile
     ),
     !!!contents,
-    sidebar_dependency()
+    sidebar_dependency(),
+    if (!identical(sidebar$open, "always")) sidebar_init_js()
```

Comment:
```suggestion
    if (sidebar_init) sidebar_init_js()
```

Reply:
Maybe include unconditionally?

Reply:
Done! I also have the initializer remove the initialization script now.